src/Components/Forms/src/EditContext.cs

```diff
@@ -176,7 +176,7 @@ public IEnumerable<string> GetValidationMessages(FieldIdentifier fieldIdentifier
     /// <param name="accessor">Identifies the field whose current validation messages should be returned.</param>
     /// <returns>The current validation messages for the specified field.</returns>
     public IEnumerable<string> GetValidationMessages(Expression<Func<object>> accessor)
-        => GetValidationMessages(FieldIdentifier.Create(accessor));
+        => GetValidationMessages(FieldIdentifier.Create(accessor, ShouldUseFieldIdentifiers));
```

Comment:
For back-compat, we'll need to eliminate this later, but it's fine for Preview 6. Same on the change below.